app/src/main/res/layout/welcome_fragment.xml

```diff
@@ -50,14 +50,31 @@
         android:layout_width="match_parent"
         android:layout_height="20dp"
         android:layout_marginBottom="32dp"
-        app:layout_constraintBottom_toTopOf="@+id/next"
+        app:layout_constraintBottom_toTopOf="@+id/knowMore"
```

Comment:
@ioane-bendingspoons I fixed some margins in the layout to be aligned with the design.